### ISSUE TITLE
Specify a link for the repo

### ISSUE THREAD

**Issue by brettcannon** [python-trio/trio#363]
I noticed that there's no link next to the repo's title, e.g. https://pypi.org/project/trio/ . What is the preferred URL to point people at for Trio? And if the answer isn't "this repo", then it probably wouldn't hurt to specify that link.

**Comment by njsmith** [comment]
Yeah, I'm not actually sure what the preferred link is :-). I've had this problem with figuring out what link to put on slides, too. This repo's README is a reasonable starting place. Or https://trio.readthedocs.io. Or the pypi.org link's not bad, but it just points back to those two places. What do you think?

**Comment by brettcannon** [comment]
I have the exact same problem you do for projects. 😄 Whichever page it is, it should have links to the other relevant pages. Since the PyPI link is more of a permalink thing then maybe that?
